include/stdint.h

```diff
@@ -38,161 +38,47 @@
  * $FreeBSD$
  */
 
+#include_next <stdint.h>
```

Comment:
Probably belongs inside the include guard. Compilers are sometimes smart about recognising include guard patterns, and this might slightly break some of their optimisations unnecessarily by being outside the guard.